plugins/inputs/snmp/snmp.go

```diff
@@ -535,8 +538,8 @@ func (t Table) Build(gs snmpConnection, walk bool) (*RTable, error) {
 		Time: time.Now(), //TODO record time at start
 		Rows: make([]RTableRow, 0, len(rows)),
 	}
-	for _, r := range rows {
-		if len(r.Tags) < tagCount {
+	for i, r := range rows {
+		if len(r.Tags)+emptyTags[i] < tagCount {
 			// don't add rows which are missing tags, as without tags you can't filter
 			continue
 		}
```

Comment:
Effectively this change is saying that adding measurements that have no tags is OK. If were going to make that change, this check (and other related code above) should just go away. Meaning if we allow tagless measurements into influxdb, we should allow them no matter the cause, whether it was because snmp was configured for a tag that ended up being empty and thus omitted, or whether it just wasn't configured with any tags at all.